bindings/python/src/bin/stub_gen.rs

```diff
@@ -0,0 +1,8 @@
+use pyo3_stub_gen::Result;
+
+fn main() -> Result<()> {
+    // `stub_info` is a function defined by `define_stub_info_gatherer!` macro.
+    let stub = _opendal::stub_info()?;
+    stub.generate()?;
```

Comment:
Can we just put in `build.rs`?

Reply:
tried that but doesn't work. `_opendal` crate is needed.

https://github.com/Jij-Inc/pyo3-stub-gen?tab=readme-ov-file#generate-a-stub-file